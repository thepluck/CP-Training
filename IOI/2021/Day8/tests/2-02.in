13 20
4 7
5 6
2 1
4 3
12 10
2 9
7 13
5 4
2 8
3 2
1 10
11 9
20
1 11 19
1 3 1
1 13 1
1 2 20
2 13
1 12 0
1 7 0
2 2
1 5 0
2 11
1 13 0
2 3
1 8 16
2 5
1 9 4
2 12
1 2 20
2 8
2 2
1 12 6
