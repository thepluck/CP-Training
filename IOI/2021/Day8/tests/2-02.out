225792840
12
6
1
1
6
6
4
4
2
85008
85008
888030
210
495
5
5
5
101745
3677355
97240
